scripts/simtest/cargo-simtest

```diff
@@ -87,35 +85,22 @@ if ! cargo nextest --help > /dev/null 2>&1; then
   exit 1
 fi
 
-if [ -n "$1" ]; then
-  echo "Running test $1"
-  CARGO_COMMAND=(nextest run "$1" --cargo-profile simulator --run-ignored all)
-else
-  echo "Running all simtest"
-  # Adding `simtest` after `run` since all walrus simtest is prefixed with `simtest`.
-  CARGO_COMMAND=(nextest run simtest --cargo-profile simulator --run-ignored all)
-fi
```

Comment:
Important: These changes should be reverted/adjusted: We have special handling of simtests where we only run those that start with the `simtest` prefix.

Reply:
Thanks, will jump on that.